Gemfile

```diff
@@ -1,11 +1,11 @@
-source "https://rubygems.org/"
+source 'https://rubygems.org/'
```

Comment:
Can you remove these changes? It looks like StringLiterals is set to `false` anyway, it shouldn't matter right?

(In any case, I'd rather enforce always-double-quote).